frontend/controller/actions/identity-kv.js

```diff
@@ -82,14 +82,17 @@ export default (sbp('sbp/selectors/register', {
       }
     })
   },
-  'gi.actions/identity/kv/setChatRoomReadUntil': ({ contractID, messageHash, createdHeight }: {
-    contractID: string, messageHash: string, createdHeight: number
+  'gi.actions/identity/kv/setChatRoomReadUntil': ({ contractID, messageHash, createdHeight, forceUpdate = false }: {
+    contractID: string,
+    messageHash: string,
+    createdHeight: number,
+    forceUpdate: boolean // In a rare case, the 'readUntil' value needs to be set to the msg with lower 'createdHeight'. (reference: https://github.com/okTurtles/group-income/issues/2729)
```

Comment:
This doesn't really say much other than that it's a 'rare case', which isn't very useful or informative. (Yes, I know the issue is referenced, but that requires extra steps and the issue also doesn't explain anything about `forceUpdate`).

If you'll add a comment here, can you say what the 'rare case' is, or if you want to be more general, say, 'in some (rare) cases, such as when the latest message is deleted, ...'? That gives immediate context to someone reading the code.

Also, the line is pretty long and the comment would be better placed above 89, possibly split across multiple lines.

Reply:
@corrideat valid point.